tests/unit/containers/my-offers/my-offers-card/MyOffersCard.spec.jsx

```diff
@@ -0,0 +1,33 @@
+import { vi } from 'vitest'
+
```

Comment:
remove this space or organize your imports

Reply:
fixed